nca/FWRules/ConnectivityGraph.py

```diff
@@ -7,13 +7,40 @@
 import re
 from collections import defaultdict
 import networkx
-from nca.CoreDS.Peer import IpBlock, ClusterEP, Pod
+from nca.CoreDS.Peer import Peer, IpBlock, PeerSet, ClusterEP, Pod
+from nca.CoreDS.ConnectionSet import ConnectionSet
+from nca.CoreDS.ProtocolSet import ProtocolSet
+from nca.CoreDS.TcpLikeProperties import TcpLikeProperties
 from .DotGraph import DotGraph
 from .MinimizeFWRules import MinimizeCsFwRules, MinimizeFWRules
 from .ClusterInfo import ClusterInfo
 
 
-class ConnectivityGraph:
+class ConnectivityGraphPrototype:
```

Comment:
can remove `ConnectivityGraphPrototype` and use only `ConnectivityGraph`?

Reply:
Done